### PR TITLE
Add tsv with filename, start, and end time for individual clips

### PR DESCRIPTION
**Pull request by adswa**
Feel free to suggest alternatives for titles, formats, etc. This file is meant to be ingested by something like ffmpeg to act on the source data video streams.

```
$ while read -r source filename start end; 
    do echo ffmpeg -i "$source" -ss "$start" -to "$end" -c:v copy -c:a copy "$filename";
    done < distribits2024/code/cuts.tsv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 00:07:05 -to 00:32:45 -c:v copy -c:a copy distribits2024/2024-04-04_1_welcome_day1.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 00:32:47 -to 00:56:47 -c:v copy -c:a copy distribits2024/2024-04-04_2_halchenko_datalad_ecosystem.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 00:56:57 -to 01:18:20 -c:v copy -c:a copy distribits2024/2024-04-04_3_hess_gitannex_complete.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 01:18:35 -to 01:42:00 -c:v copy -c:a copy distribits2024/2024-04-04_4_hanke_datalad_beyond_git.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 00:42:28 -to 02:09:38 -c:v copy -c:a copy distribits2024/2024-04-04_5_discussion_halchenko_hanke_hess.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 02:37:30 -to 02:58:05 -c:v copy -c:a copy distribits2024/2024-04-04_6_hardcastle_openneuro.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 02:58:15 -to 03:17:00 -c:v copy -c:a copy distribits2024/2024-04-04_7_thoenissen_microscopy.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 03:17:30 -to 03:30:42 -c:v copy -c:a copy distribits2024/2024-04-04_8_discussion_hardcastle_thoenniessen.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 05:08:05 -to 05:29:30 -c:v copy -c:a copy distribits2024/2024-04-04_9_poline_neurobagel_nipoppy.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 05:29:34 -to 05:49:35 -c:v copy -c:a copy distribits2024/2024-04-04_10_opiola_onedata.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 05:49:56 -to 06:07:17 -c:v copy -c:a copy distribits2024/2024-04-04_11_discussion_poline_opiola.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 06:37:32 -to 06:49:32 -c:v copy -c:a copy distribits2024/2024-04-04_12_martinez_workflow_scheduling.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 06:49:32 -to 07:01:51 -c:v copy -c:a copy distribits2024/2024-04-04_13_breuer_optimization_in_network_engineering.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 07:02:15 -to 07:12:16 -c:v copy -c:a copy distribits2024/2024-04-04_14_wexler_reproman.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 07:12:45 -to 07:23:33 -c:v copy -c:a copy distribits2024/2024-04-04_15_hoffstaedter_hpc.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-1_full.mp4 -ss 07:23:35 -to 07:40:01 -c:v copy -c:a copy distribits2024/2024-04-04_16_discussion_martinez_breuer_wexler_hoffstaedter.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 00:11:00 -to 00:23:39 -c:v copy -c:a copy distribits2024/2024-04-05_1_welcome_day2.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 00:23:39 -to 00:45:19 -c:v copy -c:a copy distribits2024/2024-04-05_2_kosciessa_neuroscientific_datamanagement.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 00:45:45 -to 01:06:18 -c:v copy -c:a copy distribits2024/2024-04-05_3_buechau_annexallthethings.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 01:06:45 -to 01:27:50 -c:v copy -c:a copy distribits2024/2024-04-05_4_discussion_kosciessa_buechau.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 02:18:40 -to 02:40:30 -c:v copy -c:a copy distribits2024/2024-04-05_5_sanders_gitannex_recipes.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 02:40:38 -to 02:58:31 -c:v copy -c:a copy distribits2024/2024-04-05_6_to_registry.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 02:58:35 -to 03:21:48 -c:v copy -c:a copy distribits2024/2024-04-05_7_discussion_pfarr_sanders_to.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 04:56:15 -to 05:35:30 -c:v copy -c:a copy distribits2024/2024-04-05_8_brechtel_kaluza_data_science_orchestrator.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 05:35:40 -to 05:56:25 -c:v copy -c:a copy distribits2024/2024-04-05_9_meyer_magit.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 05:56:25 -to 06:18:12 -c:v copy -c:a copy distribits2024/2024-04-05_10_discussion_brechtel_kaluza_meyer.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 06:46:35 -to 07:35:08 -c:v copy -c:a copy distribits2024/2024-04-05_11_durbin_range_bertuch_dataverse.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 07:35:08 -to 08:04:58 -c:v copy -c:a copy distribits2024/2024-04-05_12_unconference.mkv
ffmpeg -i sourcedata/year-2024/year-2024_day-2_full.mp4 -ss 08:04:58 -to 08:11:45 -c:v copy -c:a copy distribits2024/2024-04-05_13_farewell.mkv
```

### CODE REVIEW

**Comment by joeyh** [comment]
I had to run git-annex fix for the renamed source files to be accessible, and weirdly the command did not work when run in my usual bash shell, but did when I started "sh" and ran it in there.

File: distribits2024/clips.tsv [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+2024-04-04_00_livestream.mp4	Distribits 2024	CC-BY-3.0	2024-04-04	04	01:18:35	01:42:00	hanke_datalad_beyond_git	Michael Hanke	DataLad beyond Git, connecting to the rest of the world	DataLad has been built on Git and git-annex as foundational pillars. However, the vast majority of data infrastructures are not Git-aware. Git-annex can work with a much broader array of services, but the need to "keep the Git repo somewhere" imposes undesirable technical and procedural complexity on users. In this talk I illustrate existing means to take Git-based DataLad datasets to places that Git cannot reach on its own. Moreover, I introduce ongoing work that aims to enable DataLad users to consume non-DataLad resources as native DataLad datasets, and non-DataLad users to consume DataLad resources without DataLad, git-annex, or even Git.
+2024-04-04_00_livestream.mp4	Distribits 2024	CC-BY-3.0	2024-04-04	05	00:42:28	02:09:38	discussion_halchenko_hanke_hess	Yaroslav Halchenko, Joey Hess, Michael Hanke, Stephan Heunis	Questions and panel discussion
```
Comment (adswa) [inline distribits2024/clips.tsv]:
```suggestion
2024-04-04_00_livestream.mp4	Distribits 2024	CC-BY-3.0	2024-04-04	05	01:42:28	02:09:38	discussion_halchenko_hanke_hess	Yaroslav Halchenko, Joey Hess, Michael Hanke, Stephan Heunis	Questions and panel discussion	
```

Comment (mih) [reply distribits2024/clips.tsv]: Solved, thanks!

**Comment by mslw** [comment]
Thank you for the work in choosing timestamps and metadata properties, this makes any processing so much easier. I will merge this into the main branch and continue with adding single talk videos from there. My plan is to use mkvpropedit indstead of ffmpeg to edit metadata (so that the metadata gets picked up more nicely by vlc / mpv) and to use webm as the container format.

**Pull request merged**